src/renderer/components/dialogs/ReactionsDialog/index.tsx

```diff
@@ -81,19 +84,26 @@ function ReactionsDialogList({ reactionsByContact }: Props) {
     resolveContacts()
   }, [accountId, reactionsByContact])
 
+  const contactClickGen = (contactId: number) => {
+    return () => {
+      openViewProfileDialog(accountId, contactId)
+    }
+  }
+
   return (
     <ul className={styles.reactionsDialogList}>
       {contacts.map(contact => {
         return (
-          <li key={contact.id} className={styles.reactionsDialogListItem}>
+          <li
+            key={contact.id}
+            className={styles.reactionsDialogListItem}
+            onClick={contactClickGen(contact.id)}
+          >
             <div className={styles.reactionsDialogAvatar}>
               <AvatarFromContact contact={contact} />
             </div>
             <div className={styles.reactionsDialogContactName}>
-              <ContactName
-                displayName={contact.displayName}
-                address={contact.address}
-              />
+              <ContactName displayName={contact.displayName} />
```

Comment:
What's the point of removing the address? I don't think it's better.

Reply:
That's the original issue, I guess we need to discuss that with @r10s. While on this, when I look at the new version (if we remove email), maybe make the name bigger?

Reply:
i commented below

Reply:
for size of the name: it is fine. and re-using the same style and size as in other contact list makes sense as well.